packages/app-project/docker-compose.yml

```diff
@@ -2,31 +2,41 @@ version: '3.4'
 
 services:
   dev:
-    image: front-end-monorepo_dev:latest
+    image: front-end-monorepo_project-app:local
     build:
-      context: ../../
-      target: bootstrap
-    entrypoint:
-      - "yarn"
-      - "workspace"
-      - "@zooniverse/fe-project"
-    command: ["dev"]
+      context: .
```

Comment:
Same comment as above. The `app-project` Dockerfile works for a production deploy, but can't be relied upon for a local dev build.

Reply:
Agree - we can revist how these work for local dev.

FWIW the `FROM zooniverse/front-end-monorepo:latest` is the built image from the `bootstrap` stage in the base image. It runs `install` and `bootstrap` which install the app dependencies for resuse - https://github.com/lerna/lerna/tree/master/commands/bootstrap

Reply:
The bootstrap runs [`bootstrap.sh` here](https://github.com/zooniverse/front-end-monorepo/blob/master/bin/bootstrap.sh) too, so I think that image is also building the libraries.

Reply:
Thanks for pointing that bootstrap.sh out. 

I'm trying to come to grips with the various build tools we're using in the FEM, lerna and yarn and how these are working together for reuse across the apps.

Reply:
I think we're all still trying to come to grips with the build tools.